lib/elixir/lib/protocol.ex

```diff
@@ -195,7 +195,7 @@ defmodule Protocol do
 
   In order to speed up dispatching in production environments, where
   all implementations are known up-front, Elixir provides a feature
-  called protocol consolidation. Consolidation directly links protocols
+  called _protocol consolidation_. Consolidation directly links protocols
```

Comment:
We don’t use this notation in Elixir docs. Could you please use asterisks for italics instead? Thanks!

Reply:
Pushed!